variables.tf

```diff
@@ -276,6 +276,7 @@ variable "require_symbols" {
 variable "require_uppercase" {
   type        = "string"
   description = "Whether you have required users to use at least one uppercase letter in their password"
+  default     = ""
```

Comment:
Can we set a default value? I would vote for False, but I don't know which AWS puts in here.

Reply:
That variable was as required by a typing error, true or false can be passed as a value if it is used